assets/admin/editor-wizard/steps/lesson-patterns-step.js

```diff
@@ -19,23 +18,13 @@ import { EXTENSIONS_STORE } from '../../../extensions/store';
  * @param {Object} props.wizardData Wizard data.
  */
 const LessonPatternsStep = ( { wizardData, ...props } ) => {
-	const { senseiProExtension } = useSelect(
-		( select ) => ( {
-			senseiProExtension: select(
-				EXTENSIONS_STORE
-			).getSenseiProExtension(),
-		} ),
-		[]
-	);
-
 	const replaces = {};
 
 	if ( wizardData.title ) {
 		replaces[ 'sensei-content-title' ] = wizardData.title;
 	}
 
-	const isSenseiProActivated =
-		! senseiProExtension || senseiProExtension.is_activated === true;
+	const isSenseiProActivated = useHideEditorWizardUpsell();
```

Comment:
Could we update the variable naming to match the logic? So instead of `isSenseiProActivated` maybe we use `shouldHideEditorWizardUpsell`? (Also in the Course wizard below)

Reply:
Sure, fixed in [6ef29d3](https://github.com/Automattic/sensei/pull/5282/commits/6ef29d3d2a6668a05766de7cc31be8f354d51480).